cms/djangoapps/contentstore/views/course.py

```diff
@@ -547,7 +548,13 @@ def course_listing(request):
     optimization_enabled = GlobalStaff().has_user(request.user) and \
         WaffleSwitchNamespace(name=WAFFLE_NAMESPACE).is_enabled(u'enable_global_staff_optimization')
 
-    org = request.GET.get('org', '') if optimization_enabled else None
+    # org = request.GET.get('org', '') if optimization_enabled else None
```

Comment:
This was the original code, I have commented it out for potential future need.

Reply:
Keep this line as it is and use `get_enabled_organizations` from `openedx/features/edly/utils.py` (https://github.com/edly-io/edx-platform/blob/develop-multisite/openedx/features/edly/utils.py#L93-L110) as we are using this method for restricting course/library creation to only linked edx organizations in studio:
```python
org = request.GET.get('org', '') if optimization_enabled else None

enabled_organizations = get_enabled_organizations(request)
org = enabled_organizations[1] if enabled_organizations else None
```

Reply:
why are we doing `enabled_organizations[1]` ?

Reply:
Since the method `get_enabled_organizations` we wrote for Studio organization dropdown filter returns a list.

Reply:
yes but why not `enabled_organizations[0]`?

Reply:
Oh sorry, my bad. you are right `enabled_organizations[0]`

Reply:
let me do it

Reply:
I have added it.